### ISSUE TITLE
String output with --format=json due no errors

### ISSUE THREAD

**Issue by Iyozhikov** [twolfson/restructuredtext-lint#25]
I faced an issue when linter returned string message in case of no errors with --format=json paraeters passed.
I attached patch which fix this issue.

**Comment by twolfson** [comment]
I don't see a patch attached to this issue =/

**Comment by Iyozhikov** [comment]
Hi, sorry, i thought that it was attached - https://github.com/Iyozhikov/linter-restructured/blob/master/rst-linter.patch

**Comment by twolfson** [comment]
Ah, thanks. I will take a look at landing that by the end of the weekend (and maybe adding a test case).

**Comment by twolfson** [comment]
Taking a look at landing this now

**Comment by twolfson** [comment]
For self-reference, I applied your patch via:

```bash
# Retrieve patch
wget https://raw.githubusercontent.com/Iyozhikov/linter-restructured/master/rst-linter.patch

# Apply and stage changes
git apply rst-linter.patch
git add -p

# Perform commit based on
# https://github.com/Iyozhikov/linter-restructured/commit/14a9a72147c9684103418ade7c1c2ab34b81340f
# https://api.github.com/repos/Iyozhikov/linter-restructured/git/commits/14a9a72147c9684103418ade7c1c2ab34b81340f
git commit -m "Resolve issue with string output during json format used" --author "Igor Yozhikov <iyozhikov@mirantis.com>" --date "2015-12-09T12:22:41Z"
```

**Comment by twolfson** [comment]
This has been landed/released in `0.14.0`. Thanks for the patch =)
